src/browser/modules/D3Visualization/lib/visualization/components/layout.ts

```diff
@@ -115,15 +91,43 @@ const layout: AvailableLayouts = {
         }
         circularLayout(nodes, center, radius)
 
-        return d3force
+        simulation
           .nodes(nodes)
-          .links(relationships)
-          .size(size)
-          .start()
+          .force(
+            'collide',
+            forceCollide<VizNode>().radius(node => node.radius + 25)
+          )
+          .force(
+            'link',
+            forceLink<VizNode, Relationship>(relationships)
+              .id(node => node.id)
+              .distance(
+                relationship =>
+                  relationship.source.radius +
+                  relationship.target.radius +
+                  linkDistance
+              )
+          )
+          .force('center', forceCenter(center.x, center.y))
+          // Centering forces for nodes with no relationships to prevent them
+          // from getting pushed out of view by the forceManyBody force.
```

Comment:
Change this comment or remove it now when all nodes have the same centering forces.